### DISCUSSION TITLE
Changes in Entso-E services affect price data updates of Arska - update required

### DISCUSSION THREAD

**Discussion by Olli69** [Netgalleria/arska-node#27] [Announcements]
Hi,
We have just noticed that server address of Entso-E API has been changed today and the address used by Arska is not working any more. 

We made a quick fix update **0.93.0-beta3** which you can download from Arska install page https://iot.netgalleria.fi/arska-install/ or you can do launch automatic update from Arska (Admin/Firmware Admin). Remenber to backup your Arska configuration before the update. Please let me know if there are any problems with the update.

If you have build your own version change server name in main.cpp:
```
//this is old definition, comment out or remove following line:
// const char *host_prices PROGMEM = "transparency.entsoe.eu"; //!< EntsoE reporting server for day-ahead prices
//this is new url for Entso-E API
const char *host_prices PROGMEM = "web-api.tp.entsoe.eu";
```

Information about the change on Entso-E web site:  see more https://transparency.entsoe.eu/news/widget?id=63eb9d10f9b76c35f7d06f2e

**Comment by joe-ave** [comment]
Hello Olli,

thank you!, I edited and recompiled my local version and now it works again.
It did not work at first, I ended up erasing the flash and after that uploading filesystem and then the general upload, then it started to work after a couple of minutes.

//Jonas

**Reply by stikstone** [reply]
Started to work, But I get error message 
"03:06:29 (+1) Cannot get price info for current period."'
Does not seem to affect anything, but?

**Reply by Olli69** [reply]
Arska has very simple error logging system,  which shows history errors even there is no error state any more. If there is an error for example in the startup, it stays visible on the main screen until you acknowledge it (check  box) or if there is an new error. So, this was probably a random error in the startup (time was not synchronised) .
